web/shared/api.ts

```diff
@@ -13,22 +13,36 @@ export const removeUnauthorizedCallback = authMiddleware.removeUnauthorizedCallb
 export function deleteSession(streamId: string, clientId: string) {
     return w.url(`/session/${streamId}/${clientId}`).delete().res();
 }
+export function getCurrentNode(): string {
+    const urlParams = new URLSearchParams(window.location.search);
+    return urlParams.get('nodes') || '0';
```

Comment:
nodes name is set in `alias = "static-0"`

`nodes=0` is only developer mode. https://github.com/binbat/live777/tree/main/conf/livenil